__tests__/app.test.js

```diff
@@ -12,27 +13,45 @@ describe('/api/topics', () => {
         return request(app)
         .get('/api/topics')
         .expect(200)
-        .then((response) => {
-            expect(response.body.topics).toHaveLength(3)
-            response.body.topics.forEach((topic) => {
-                expect(typeof topic.description).toBe('string')
-                expect(typeof topic.slug).toBe('string')    
+        .then(({ body }) => {
+          expect(body.topics).toHaveLength(3)
+          body.topics.forEach((topic) => {
+            expect(topic).toHaveProperty('slug')
+            expect(topic).toHaveProperty('description')
+          })
+        })
+    })
+})
+
+describe('/api endpoint', () => {
+    describe("error handling", () => {
+        test("404 for missing endpoints", () => {
+          return request(app)
+          .get('/api/missing')
+          .expect(404)
+          .then(({body}) => {
+            expect(body.msg).toEqual('/api/missing endpoint not found')
             })
         })
     })
 })
-describe("error handling", () => {
-    test("404 for missing endpoints", () => {
-      return request(app)
-      .get('/api/missing')
-      .expect(404)
-      .then(({body}) => {
-        expect(body.msg).toEqual('/api/missing endpoint not found')
+
+describe('/api endpoint', () => {
+    test('GET: 200 should return an object describing all available endpoints', () => {
```

Comment:
No need to use FS here. We are just accessing a local file from our repo, but by using fs you have made this part of the process asynchronous unnecessarily. You can simply require the JSON in as you would any other local file